src/redux/todoActions.tsx

```diff
@@ -0,0 +1,6 @@
+export const addTodo = (data: string) => {
```

Comment:
For now it's ok, but I think data would be TodoModal

Reply:
Updated and used TodoModal